### PR TITLE
Fix/Developer Guide and SampleDataUtil

### PR DESCRIPTION
**Pull request by vigonometry**
## Development
- Updated SampleDataUtil to be relevant to Docedex

## Developer Guide
- Updated formatting of sections
- Ensured proper rendering of Table of Contents

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -8,37 +8,60 @@
...
-            new Person(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
-                getTagSet("neighbours")),
-            new Person(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
-                getTagSet("family")),
-            new Person(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@example.com"),
-                getTagSet("classmates")),
-            new Person(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@example.com"),
-                getTagSet("colleagues"))
+
+    public static Doctor[] getSampleDoctors() {
```
Comment (praveenkrishna0512) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:28]: I believe sampleDoctors and sampleAddressBook is not used. Do we want to remove it completely?

**Review by praveenkrishna0512 — COMMENTED** [review]
Left a few coments!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -34,44 +31,36 @@ than current GUI apps in the industry.
...
 - For Windows Users:
   - Double-click `docedex.jar` to launch the application.
 - If this still does not work follow the instructions listed [here.](#launching-docedex)
 - You should notice the GUI of the application pop up.
-  <img src="images/NewUi.png" alt="NewUi" width="70%" height="70%">
-  <br><br><br>
 5. Learn more about navigating the GUI [here](#navigating-the-graphical-user-interface-gui).
-  <br><br><br>
 6. For **new users**, learn to use Docedex [here](#docedex-tutorial-for-new-users).
-  <br><br><br>
```
Comment (praveenkrishna0512) [inline docs/UserGuide.md:67]: Any reason for removing the line breaks?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -34,44 +31,36 @@ than current GUI apps in the industry.
...
 2. Download the latest jar file (`docedex.jar`) from our [Github Releases](https://github.com/AY2223S2-CS2103T-F12-1/tp/releases/).
    ![DownloadDocedex](images/DownloadDocedex.png)
-   <br><br><br>
 3. Move the jar file to a folder where you want to store the data of the doctors and patients.
 - Create a new folder called (you can name it Docedex) where you would like to house the application (e.g., Desktop, Documents, etc.).
 - Move the downloaded jar file to this folder as shown below.
-  - For Mac Users:<br>
-  <img src="images/DownloadDocedexMac.png" alt="DownloadDocedexMac" width="70%" height="70%">
-  - For Windows Users:<br>
-  <img src="images/DocedexInFileWin.png" alt="DocedexInFileWin" width="70%" height="70%">
```
Comment (praveenkrishna0512) [inline docs/UserGuide.md:51]: I put the size restrictions on the images as I felt that they were too big. Do let me know if you had any considerations in removing them!

File: docs/UserGuide.md (hunk 2 above)
Comment (vigonometry) [reply docs/UserGuide.md:67]: Improve readability on print

File: docs/UserGuide.md (hunk 3 above)
Comment (vigonometry) [reply docs/UserGuide.md:51]: To improve visibility of images and readability of embedded content (on print, the images were too small and the text within was very hard to read).

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 4]
```diff
@@ -51,33 +49,6 @@ public static Index parseIndex(String oneBasedIndex) throws ParseException {
...
-        if (!StringUtil.isNonZeroUnsignedInteger(secondIndex)) {
-            throw new ParseException(MESSAGE_INVALID_INDEXES);
-        }
-
-        List<Index> returnList = new ArrayList<>();
-        returnList.add(Index.fromOneBased(Integer.parseInt(firstIndex)));
-        returnList.add(Index.fromOneBased(Integer.parseInt(secondIndex)));
-        return returnList;
-    }
-
```
Comment (owen-yap) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:80]: Wait why are we removing this? Seems important

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 1 above)
Comment (vigonometry) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: Removed post discussion

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 4 above)
Comment (vigonometry) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java:80]: Resolved post discussion:
- Assign and Unassign Command mechanisms were updated and made this function redundant
- No usages of this function was found on IntelliJ

**Review by praveenkrishna0512 — COMMENTED** [review]
Waiting upon minor cosmetic changes

**Review by praveenkrishna0512 — APPROVED** [review]
LGTM!

**Pull request merged**